ApiBundle/Controller/AreaController.php

```diff
@@ -154,16 +154,19 @@ public function deleteAreaFromTemplateAction($areaId, $templateId)
      */
     public function deleteAreaFromAreaAction($areaId, $parentAreaId, $nodeId = null, $templateId = null)
     {
+        $areaContainer = null;
+
         if ($nodeId) {
             $nodeRepository = $this->get('open_orchestra_model.repository.node');
             $node = $nodeRepository->find($nodeId);
             $areaContainer = $nodeRepository->findAreaByAreaId($node, $parentAreaId);
-        }
-        if ($templateId && is_null($nodeId)) {
+        } else if ($templateId && is_null($nodeId)) {
```

Comment:
the second part of the test is not usefull I think.

If nodeId is not null, it will go directly in the if, and not in the elseif